docs/source/remote_data.rst

```diff
@@ -79,7 +79,7 @@ Historical dividends from Yahoo! Finance.
 .. _remote_data.yahoo_quotes:
```

Comment:
there is also a TOC in this file (its done manually but that's ok). add an entry there as well.